apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregningsgrunnlag/YrkesskadeTrygdetidOMS.tsx

```diff
@@ -0,0 +1,82 @@
+import { BodyShort } from '@navikt/ds-react'
+import { useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import styled from 'styled-components'
+import { ApiErrorAlert } from '~ErrorBoundary'
+import { LovtekstMedLenke } from '~components/behandling/soeknadsoversikt/soeknadoversikt/LovtekstMedLenke'
+import Spinner from '~shared/Spinner'
+import { ITrygdetid, hentTrygdetid, opprettTrygdetid, lagreYrkesskadeTrygdetidGrunnlag } from '~shared/api/trygdetid'
+import { isFailure, isPending, useApiCall } from '~shared/hooks/useApiCall'
+
+const YrkesskadeTrygdetid = () => {
+  const { behandlingId } = useParams()
+  const [hentTrygdetidRequest, fetchTrygdetid] = useApiCall(hentTrygdetid)
+  const [opprettTrygdetidRequest, requestOpprettTrygdetid] = useApiCall(opprettTrygdetid)
+  const [opprettYrkesskadeTrygdetidGrunnlag, requestOpprettYrkesskadeTrygdetidGrunnlag] = useApiCall(
+    lagreYrkesskadeTrygdetidGrunnlag
+  )
+
+  useEffect(() => {
+    if (!behandlingId) throw new Error('Mangler behandlingsid')
+
+    fetchTrygdetid(behandlingId, (trygdetid: ITrygdetid) => {
+      if (trygdetid == null) {
+        requestOpprettTrygdetid(behandlingId, () => {
+          requestOpprettYrkesskadeTrygdetidGrunnlag({ behandlingsId: behandlingId })
+        })
+      } else {
+        // Må skrives om når vi gjør om til å støtte utenlands og poeng i inn/ut år (relatert til prorata)
+        // Pt sjekker vi regelResultat string i backend (Trygdetid.kt) for å gjøre en "er dette yrkesskade"
+        // I mellomtid så vil den bare erstatte en fast 40 år med en ny fast 40 år på samme grunnlag
+        requestOpprettYrkesskadeTrygdetidGrunnlag({ behandlingsId: behandlingId })
+      }
+    })
+  }, [])
+
+  return (
+    <TrygdetidWrapper>
+      <LovtekstMedLenke
+        tittel={'Trygdetid'}
+        hjemler={[
+          {
+            tittel: '§ 17-12.Pensjon etter dødsfall som skyldes yrkesskade',
+            lenke: 'https://lovdata.no/lov/1997-02-28-19/§17-12',
+          },
+        ]}
+        status={null}
+      >
+        <TrygdetidInfo>
+          <BodyShort>Dødsfall som skyldes en skade eller sykdom som går inn under kapittel 13</BodyShort>
+          <BodyShort>
+            Trygdetid: <strong>40 år</strong>
+          </BodyShort>
+        </TrygdetidInfo>
+      </LovtekstMedLenke>
+
+      {isPending(hentTrygdetidRequest) && <Spinner visible={true} label={'Henter trygdetid'} />}
```

Comment:
Hvorfor skal vi hente den om den er hardkodet til 40?

Reply:
Forsto det sånn at den hentes for å finne ut om det er yrkesskade og derfor 40 år. Men mulig jeg har missforstått. Det er en direkte kopi av BP sin, med endret tekst for hjemmel

Reply:
Ok, merger denne og lager en ny sak på å skrive vekk hardkodingen. Da må det vel også endres tilsvarende for BP? Det er endel i denne flyten hvor kanskje litt mye av logikken ligger i frontend

Reply:
https://jira.adeo.no/browse/EY-2547